packages/adminrouter/windows.build.ps1

```diff
@@ -20,8 +19,10 @@ function Unzip {
 
 Unzip "c:\pkg\src\OpenResty\openresty-1.15.8.2-win64.zip" "c:/Temp/openresty"
 
-ls "C:\pkg\build\extra\src\"
```

Comment:
Error caused by difference of `C:\Temp` and `$TempFolder`?

Reply:
Thank you, fixed it.